### PR TITLE
pog changes

### PR DESCRIPTION
**Pull request by zAG6z**
pog

### CODE REVIEW

**Review by xf8b — APPROVED** [review]
iapprove

**Comment by zAG6z** [comment]
thx

File: pluto.js [hunk 1]
```diff
@@ -18,6 +18,11 @@ const client = new Discord.Client();
 // login
 client.login(auth.token);
 
+//secret addon thats useless
+client.on('ready', () =>{
+    console.log('putin is good')
+    )};
```
Comment (xf8b) [inline pluto.js:24]: shouldnt this be ```js
client.on('ready', () => {
    console.log('putin is good');
});
```

**Review by xf8b — CHANGES_REQUESTED** [review]
actually nvm you needa change some stuff

File: pluto.js (hunk 1 above)
Comment (zAG6z) [reply pluto.js:24]: idk

Comment (zAG6z) [reply pluto.js:24]: i dont think u need the ;

Comment (zAG6z) [reply pluto.js:24]: i need a pfp

Comment (xf8b) [reply pluto.js:24]: eh i am java dev and i like semicolons

Comment (xf8b) [reply pluto.js:24]: but i dont think
```js
client.on('ready', () => {
    console.log('putin is good')
)};
``` 
is correct

Comment (zAG6z) [reply pluto.js:24]: fine ill have a look in vsc then twat

Comment (xf8b) [reply pluto.js:24]: lmao

Comment (zAG6z) [reply pluto.js:24]: it does work u prick

Comment (xf8b) [reply pluto.js:24]: bruh ok

Comment (xf8b) [reply pluto.js:24]: you sure you typed it exactly as you put it

Comment (zAG6z) [reply pluto.js:24]: yes

Comment (xf8b) [reply pluto.js:24]: ok then

Comment (zAG6z) [reply pluto.js:24]: do you want the fuckin code

**Review by xf8b — APPROVED** [review]
iapprove part 2

File: pluto.js (hunk 1 above)
Comment (zAG6z) [reply pluto.js:24]: `client.on('ready', () => {
    console.log("putin is pog")
});`

Comment (zAG6z) [reply pluto.js:24]: idfk how to work github code shit

Comment (xf8b) [reply pluto.js:24]: thats not the same as what i put

Comment (xf8b) [reply pluto.js:24]: i said that 
```js
client.on('ready', () =>{
    console.log('putin is good')
    )};
```
might be wrong not 
```js
client.on('ready', () =>{
    console.log('putin is good')
    });
```

Comment (zAG6z) [reply pluto.js:24]: THERES NO DIFFERENCE

Comment (zAG6z) [reply pluto.js:24]: OMFG

Comment (xf8b) [reply pluto.js:24]: there is
one has )} and the other has })

Comment (zAG6z) [reply pluto.js:24]: WELL IT WORKS SO BE QUIET AND APPROVE IT AGAIN

Comment (xf8b) [reply pluto.js:24]: are you sure

Comment (zAG6z) [reply pluto.js:24]: OH MY FUCKIGNG GFDIJGUIYUASDBGI YES U TWAT

Comment (zAG6z) [reply pluto.js:24]: DO YOU WANT TO FUCKING LOG OUTPUT

Comment (zAG6z) [reply pluto.js:24]: image

Comment (zAG6z) [reply pluto.js:24]: THERE

Comment (zAG6z) [reply pluto.js:24]: HAPPY NIOW

Comment (xf8b) [reply pluto.js:24]: no

Comment (zAG6z) [reply pluto.js:24]: SERNJSERHGDUIF UIFSEGF RISE MFTHDESFDGFTR

Comment (zAG6z) [reply pluto.js:24]: APPROVE IT AGAIN

Comment (zAG6z) [reply pluto.js:24]: DICKHEAD

Comment (xf8b) [reply pluto.js:24]: bruh i literally approved 9 minutes ago right below

Comment (zAG6z) [reply pluto.js:24]: I NEED ANOTHJER APORVEAL

Comment (xf8b) [reply pluto.js:24]: pls stop screaming

Comment (zAG6z) [reply pluto.js:24]: why

Comment (xf8b) [reply pluto.js:24]: thanks

Comment (zAG6z) [reply pluto.js:24]: ok

**Review by xf8b — APPROVED** [review]
iapprove part 3
java mascot giving a thumbs up

**Comment by xf8b** [comment]
@secnyt pls merge

**Comment by xf8b** [comment]
:crab:

**Pull request closed without merging**